scripts/test_ec.sh

```diff
@@ -174,7 +174,7 @@ do
     fec_type=$(echo $i|cut -d_ -f1)
     word_size=$(echo $i|cut -d_ -f2)
 
-    do_test enconly ${fec_type} ${word_size} 50 50 "" ""
+#    do_test enconly ${fec_type} ${word_size} 50 50 "" ""
```

Comment:
Why not removing it totally (it will not be lost, thanks to Git history), and explain why we remove it in the commit message.